doc/source/tune/api/execution.rst

```diff
@@ -53,5 +53,6 @@ tune.run_experiments
     :toctree: doc/
 
     run_experiments
+    run
```

Comment:
We haven't deprecated this API but we are hiding this from docs since `Tuner` is the new way to do things.

Reply:
Let's just include it since we didn't explicitly deprecate. Users have asked for this API ref in the past.

Reply:
sound good